src/main/java/com/github/pires/obd/commands/ObdCommand.java

```diff
@@ -1,8 +1,7 @@
 /**
  * Licensed under the Apache License, Version 2.0 (the "License"); you may not
  * use this file except in compliance with the License. You may obtain a copy of
- * the License at
- *     http://www.apache.org/licenses/LICENSE-2.0
+ * the License at http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
hm probbaly thats the IDE's work, I usually do not touch the licensing info
:) I check whats happening

2015-07-16 21:55 GMT+02:00 Paulo Pires <notifications@github.com>:

> In src/main/java/com/github/pires/obd/commands/ObdCommand.java
> <https://github.com/pires/obd-java-api/pull/47#discussion_r34830690>:
>
> > @@ -1,8 +1,7 @@
> >  /**
> >   * Licensed under the Apache License, Version 2.0 (the "License"); you may not
> >   * use this file except in compliance with the License. You may obtain a copy of
> > - * the License at
> > - *     http://www.apache.org/licenses/LICENSE-2.0
> > + * the License at http://www.apache.org/licenses/LICENSE-2.0
>
> Licensing formatting again.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pires/obd-java-api/pull/47/files#r34830690>.
>